src/components/ProductContent/ProductMedia.tsx

```diff
@@ -1,16 +1,66 @@
 import React from 'react'
+import { useState, useEffect } from 'react'
+import cx from 'classnames'
+
+//TO DO: Retrieve images from database instead
+const srcs = [
+  '/images/nike-jordan-off-white.jpg',
+  '/images/nike-woman-running-shoe.jpg',
+  '/images/nike-air-vapormax-360-herren.jpg',
+]
+
+interface ISrc {
+  src: string
+}
```

Comment:
3 things here:
1. We have agreed to name the interfac weithout prefix `I`.
2. Seems like we don't really need to have an interface here, a simple approach will do the job.
3. Do you think `imagePath` is more meaningful than `src` ?

```suggestion
const imagePaths: string = [
  '/images/nike-jordan-off-white.jpg',
  '/images/nike-woman-running-shoe.jpg',
  '/images/nike-air-vapormax-360-herren.jpg',
]
```

Reply:
I agree with your approach, I'll try on that. Thanks.